bin/p2-bootstrap/bootstrap.go

```diff
@@ -92,7 +93,15 @@ func InstallConsul(consulPod *pods.Pod, consulManifest *pods.PodManifest) error
 	return nil
 }
 
-func VerifyConsulUp() error {
+func VerifyConsulUp(timeout string) error {
+	timeoutDur, err := time.ParseDuration(timeout)
+	if err != nil {
+		timeoutDur = 10 * time.Second
+	}
+	if timeoutDur == 0 {
+		return nil
```

Comment:
If this is going to be the behavior, we should probably be clearer about the argument name. `--timeout` could express a constraint on the entire bootstrapping process - it's not obvious that it's just applied to the consul ping. Or, we make `timeout=0` mean try infinitely and add a different argument `--skip-consul-ping` to do what you want here.